tools-reference/bash/text.xml

```diff
@@ -523,41 +365,25 @@ following are available (lifted from <c>man bash</c>):
 </p>
 
 <table>
-  <tr>
-    <th>
-      Operator
-    </th>
-    <th>
-      Purpose
-    </th>
-  </tr>
-  <tr>
-    <ti>
-      <c>file1 -nt file2</c>
-    </ti>
-    <ti>
-      file1 is newer (according to modification date) than
-      file2, or if file1 exists and file2 does not.
-    </ti>
-  </tr>
-  <tr>
-    <ti>
-      <c>file1 -ot file2</c>
-    </ti>
-    <ti>
-      file1 is older than file2, or if file2 exists and
-      file1 does not.
-    </ti>
-  </tr>
-  <tr>
-    <ti>
-      <c>file1 -ef file2</c>
-    </ti>
-    <ti>
-      file1 and file2 refer to the same device and inode
-      numbers.
-    </ti>
-  </tr>
+<tr>
+  <th>Operator</th>
+  <th>Purpose</th>
+</tr>
+<tr>
+  <ti><c>file1 -nt file2</c></ti>
+  <ti>
+    file1 is newer (according to modification date) than file2,
+    or if file1 exists and file2 does not.
+  </ti>
+</tr>
+<tr>
+  <ti><c>file1 -ot file2</c></ti>
+  <ti>file1 is older than file2, or if file2 exists and file1 does not.</ti>
+</tr>
+<tr>
+  <ti><c>file1 -ef file2</c></ti>
+  <ti>file1 and file2 refer to the same device and inode numbers.</ti>
```

Comment:
Out of curiosity, why not use the same phrasing as bash's `help test` (file1 is a hard link to file2)? I find it slightly easier to understand.

Reply:
Good point. The "inode" wording was from the GNU Info manual.